src/js/frontend/src/Redux/reducer.js

```diff
@@ -20,16 +24,37 @@ export function rootReducer (state = IMap(), action) {
 
 const defaultState = {
   showAboutScreen: false,
+  entities: {
+    people: {},
+    organisations: {},
+    "government-offices": {},
+  },
 };
 
-export function aboutReducer(state = defaultState, {type}) {
-  let newState = Object.assign({}, state);
-
+function aboutReducer(state = defaultState.showAboutScreen, {type}) {
   switch (type) {
     case actionTypes.TOGGLE_ABOUT:
-      newState.showAboutScreen = !newState.showAboutScreen;
-      return newState;
+      return ! state;
```

Comment:
Blegh, really? :stuck_out_tongue_closed_eyes:  OK then, not my personal taste but you were here first.  If you care about that then I recommend you submit a PR changing [the relevant `.eslintrc.yml` option](http://eslint.org/docs/rules/space-unary-ops) to match your preference.  Then Hound will help keep things consistent.